web/src/scripts/containers/TabbedEditor.jsx

```diff
@@ -171,6 +176,43 @@ class TabbedEditor extends Component {
       this.setState({files})
       this.openMenu(MENU_FILE_SEARCH)
     },
+    focusNextGroup: (e) => {
+      const { dispatch } = this.props
+      dispatch(tabActions.focusAdjacentGroup(CONTENT_PANES.CENTER, (i) => i+1))
```

Comment:
This function param seems non-standard to me. I think I'd normally expect `'next'` or `'prev'`